packages/react-dom-bindings/src/events/plugins/FormActionEventPlugin.js

```diff
@@ -92,6 +96,20 @@ function extractEvents(
   }
   const formInst = maybeTargetInst;
   const form: HTMLFormElement = (nativeEventTarget: any);
+
+  if (!disableInputAttributeSyncing) {
+    // eslint-disable-next-line no-for-of-loops/no-for-of-loops
+    for (const element of form.elements) {
```

Comment:
Do we iterate through DOM collections with for-of in other places as well?

Reply:
686 times the for...of construction in repo (_36 with disable-next-line_), in some places specifically for DOMCollection:
* https://github.com/facebook/react/blob/3e94bce765d355d74f6a60feb4addb6d196e3482/packages/react-devtools-extensions/src/panel.js#L15
* https://github.com/facebook/react/blob/8fbd3079425eaff85ec1b52b0eefecdd44ba7289/packages/react-devtools-extensions/src/main/cloneStyleTags.js#L5

But this is devtools, I haven't encountered such a case specifically for react-dom

Reply:
devtools has less restrictions. Do we iterate through DOM collections somewhere else in react-dom?

Reply:
Nope, "_But this is devtools, I haven't encountered such a case specifically for react-dom_"

Reply:
We don't use `for-of` in react-dom-bindings so let's just stick with a classic `for (let i ...)` loop for safety.

Reply:
✅